.eslintrc.json

```diff
@@ -6,15 +6,14 @@
     },
     "plugins": [
         "@typescript-eslint",
-        "react",
-        "react-hooks",
```

Comment:
It's not required to specify these plugins, as extending their recommended configs will automatically include them.